PocketKit/Sources/PocketKit/MyList/SearchItemsList/SkeletonView.swift

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+struct SkeletonView: View {
+    var body: some View {
+        List {
+            ForEach(0..<10) { _ in
+                HStack {
+                    VStack(alignment: .leading) {
+                        Image(asset: .itemSkeletonTitle)
+                            .resizable()
+                            .frame(height: 60)
+                        Image(asset: .itemSkeletonTags)
+                    }
+                    Spacer()
+                    VStack(spacing: 12) {
```

Comment:
should we group the magic numbers in a `private enum` or something?

Reply:
I get a `Non-constant range: argument must be an integer literal` for when I set the magic number in the `ForEach` for the 10 skeleton cells, but I added a constant for the other 2 values.